### PR TITLE
Add roles descriptions for mentoring subcommittee

### PR DESCRIPTION
**Pull request by rgaiacs**
@ChristinaLK Could you review and change it if need?

@raynamharris Could you merge this after @ChristinaLK's green light?

### CODE REVIEW

**Comment by raynamharris** [comment]
This looks good so far, but I have a few comments

1. We don't have a co-chair yet.
2. Should we add a role for the SWC board liason?
3. The protocol for using AMY to generate a list of names and emails could be added. 
4. Should we add a brief description of the roles of the non-named committee members? Even if they don't have specific tasks listed, one could list the number of hours per week spent on things related to the committee.

**Comment by ChristinaLK** [comment]
Agreed - I think this is a good starting point, but maybe file @raynamharris comments as an issue (or send to me in email) so that we do address them in the future?

**Comment by rgaiacs** [comment]
> We don't have a co-chair yet.

I thought that the co-chair could be the liaison.

> Should we add a role for the SWC board liason?

The steering committee need to vote on this.

>  The protocol for using AMY to generate a list of names and emails could be added. 

Sure.

> Should we add a brief description of the roles of the non-named committee members? Even if they don't have specific tasks listed, one could list the number of hours per week spent on things related to the committee.

Sure.

File: subcommittees/mentoring/roles.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+
+   The agenda can be created on http://pad.software-carpentry.org/.
+
+2. Chair creates calendar event for the meeting with one week in advance.
+
+   The calendar event can be created on http://calendar.google.com/.
+
+3. Secretary takes notes during the meeting.
+
+4. Secretary creates issues for action itens on GitHub [repository][].
```
Comment (wking) [inline subcommittees/mentoring/roles.md]: “itens” → “items”.  And maybe shuffle to “Secretary creates [GitHub issues][issues] for action items.” to be more compact.

[issues]: https://github.com/swcarpentry/board/issues

File: subcommittees/mentoring/roles.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+
+6. Vice-chair merges the minutes.
+
+7. Vice-chair reports meetings during Steering Committee meeting.
+
+## Apendices
+
+### Time
+
+1. Meetings need to be run at least twice on the same day to accomodate different timezones.
```
Comment (wking) [inline subcommittees/mentoring/roles.md]: “accomodate” → “accommodate”

File: subcommittees/mentoring/roles.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+
+2. Meeting time must be on UTC to be fair with everyone.
+
+### Etherpad operation
+
```
Comment (wking) [inline subcommittees/mentoring/roles.md]: This still needs to get filled in?

File: subcommittees/mentoring/roles.md [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+### Google Calendar operation
+
+When creating a new event or updating a old one.
+
+![Calendar event steps](img/calendar-event.png)
+
+1. Set the name of the event or meeting.
+
+2. Set the date. **Please use UTC time.**
+   If using Google Calendar select Ghana, Accra or Reykjavik, Iceland
```
Comment (wking) [inline subcommittees/mentoring/roles.md]: “Ghana, Accra” → “Accra, Ghana”?  Or is it really listed country-first?

File: subcommittees/mentoring/roles.md [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+   since they doesn't have daylight saving time.
```
Comment (wking) [inline subcommittees/mentoring/roles.md]: “doesn't” → “don't”

**Comment by rgaiacs** [comment]
Close by 7f140fa after some rebase and fix of dates.

File: subcommittees/mentoring/roles.md [hunk 6]
```diff
@@ -0,0 +1,111 @@
...
+The Vice Chair is meant to support the duties of the chair. In particular, the 
+Vice Chair should adopt the responsibilities of the Chair when the Chair is 
+temporarily unable to perform their duties (e.g., absent from a meeting, 
+stepping down during a vote temporarily due to conflict of interest). The Vice
+Chair agrees to:
+
+1.  Adopt the duties and authority of the Chair at the request of the Chair or
+    by vote of the Committee.
+
+2.  Act as the primary liaison between the Steering Committee and the Subcommittee.
```
Comment (karinlag) [inline subcommittees/mentoring/roles.md]: I would not include 2. The liaison may or may not be the vice chair, I don´t think this should be a given.

File: subcommittees/mentoring/roles.md [hunk 7]
```diff
@@ -0,0 +1,111 @@
...
+
+   The calendar event can be created on http://calendar.google.com/.
+
+3. Secretary takes notes during the meeting.
+
+4. Secretary creates issues for action itens on GitHub [repository][].
+
+5. Secretary creates pull request with minutes archives.
+
+6. Vice-chair merges the minutes.
```
Comment (karinlag) [inline subcommittees/mentoring/roles.md]: ...it seems more natural to me that the Secretary merges the minutes? The secretary will be more aware of the status of the minutes, re changes afterwards etc, than the vice chair.

File: subcommittees/mentoring/roles.md [hunk 8]
```diff
@@ -0,0 +1,111 @@
...
+
+7. Vice-chair reports meetings during Steering Committee meeting.
```
Comment (karinlag) [inline subcommittees/mentoring/roles.md]: Liaison reports meetings. See above regarding liaison and vice chair.

File: subcommittees/mentoring/roles.md [hunk 9]
```diff
@@ -0,0 +1,111 @@
...
+
+## Apendices
```
Comment (karinlag) [inline subcommittees/mentoring/roles.md]: Appendices

File: subcommittees/mentoring/roles.md [hunk 10]
```diff
@@ -0,0 +1,111 @@
...
+
+### Time
+
+1. Meetings need to be run at least twice on the same day to accomodate different timezones.
+
+2. Meeting time must be on UTC to be fair with everyone.
+
```
Comment (karinlag) [inline subcommittees/mentoring/roles.md]: in UTC

fair to everybody

File: subcommittees/mentoring/roles.md [hunk 11]
```diff
@@ -0,0 +1,111 @@
...
+4. Set the description.
+   **The link to the agenda goes here.**
+
+5. Set notifications.
+   Is good to have one notification on the day before and another 10 minutes before the meeting
+   as email.
+
+6. Set guests.
+   For mentoring subcommittee meetings those are all the members of the subcommittee.
+   For discussion sessions those are all the instructors that you invited.
```
Comment (karinlag) [inline subcommittees/mentoring/roles.md]: Specify where the list of members with email addresses can be found.

**Comment by rgaiacs** [comment]
@ChristinaLK Could you work on @karinlag's comments?

**Pull request closed without merging**